Makefile

```diff
@@ -128,3 +128,25 @@ terraform-plan: terraform-init
 .PHONY: terraform-destroy
 terraform-destroy: terraform-init
 	terraform -chdir=terraform/aks destroy -var-file workspace_variables/${DEPLOY_ENV}.tfvars.json ${AUTO_APPROVE}
+
+enable-maintenance:
+	$(if ${DEPLOY_ENV} == "review_aks", $(eval VARIABLE_FILE_NAME=review), $(eval VARIABLE_FILE_NAME=${DEPLOY_ENV}))
+	$(eval include terraform/workspace-variables/${VARIABLE_FILE_NAME}.tfvars)
+	$(eval SPACE=${paas_space_name})
+
+	cf target -s ${SPACE}
+	cd service_unavailable_page && cf push
+	cf map-route ecf-unavailable london.cloudapps.digital --hostname ecf-review-pr-3766
```

Comment:
should this be

`cf map-route ecf-unavailable london.cloudapps.digital --hostname ${APP_NAME}`

instead?